plugins/src/android/OptimoveSDKPlugin.java

```diff
@@ -128,11 +131,25 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
         case CLEAR_CONTEXT:
             this.clearJsContext();
             return true;
+        case CHECK_IF_PENDING_DDL_EXISTS:
+            this.checkIfPendingDDLExists(callbackContext);
+            return true;
         }
 
         return false;
     }
 
+    private void checkIfPendingDDLExists(CallbackContext callbackContext) {
+        if (jsCallbackContext == null) {
+            jsCallbackContext = callbackContext;
+        }
+        if (null != pendingDDL) {
+            OptimoveSDKPlugin.sendMessageToJs("deepLink", pendingDDL);
+            pendingDDL = null;
+            pendingActionId = null;
```

Comment:
pendingActionId is only relevant for push, it has to do with action buttons in push notifications

Reply:
I deleted it 👍

Reply:
mb didnt push? deleted also ;)